doc/py_tutorials/py_gui/py_video_display/py_video_display.markdown

```diff
@@ -25,17 +25,21 @@ import numpy as np
 import cv2 as cv
 
 cap = cv.VideoCapture(0)
-
+if not cap.isOpened:
+    print("Cannot open VideoCapture")
+    exit()
 while(True):
     # Capture frame-by-frame
     ret, frame = cap.read()
 
-    # Our operations on the frame come here
-    gray = cv.cvtColor(frame, cv.COLOR_BGR2GRAY)
+    # if frame is read correctly ret is True
+    if ret:
```

Comment:
What is about this?

```
if not ret:
    print("Can't receive frame (stream end?). Exiting...")
    break
```

At least we don't need to complicate condition below.